pkgs/applications/version-management/git-and-tools/git-appraise/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "git-appraise-unstable-${version}";
+  version = "2018-02-26";
```

Comment:
It seems like the commit was on February 27th.

Reply:
@haslersn weird… go2nix did put this :thinking: but indeed GitHub shows Feb. 27th

Reply:
ah… because of my timezone I guess
```
git show 2414523
commit 2414523905939525559e4b2498c5597f86193b61 (HEAD -> master, origin/master, origin/HEAD)
Merge: 0cd354a 7c60f9d
Author: Omar Jarjur <ojarjur@google.com>
Date:   Mon Feb 26 21:36:54 2018 -0500
```